site/tailwind.config.js

```diff
@@ -30,6 +30,21 @@ export default {
       screens: {
         md: '848px',
       },
+      keyframes: {
```

Comment:
I assume this is for test purposes?

Reply:
i wasn't sure where i need to put this to get the fade in animation to work. noticed there is a top level `tailwind.config.js` and this one. let me know if i should delete it.

Reply:
ideally those should come from OnchainKit, no?